source/aks/redeploying-aks-clusters.html.md.erb

```diff
@@ -33,7 +33,12 @@ kubectl get pods -A | wc -l > pods_total_count && kubectl get pods -A > pods_all
 
 Three files will be saved in the directory from which you are running the above command. 
 
-- Investigate failed helm releases and missing pods as required
+- Investigate failed helm releases and pods in CrashLoopBackOff Status. Check if these pods have pod disruption budgets 
+
+```command
+kubectl get pdb -n NAMESPACE
```

Comment:
```suggestion
kubectl get pdb -A
```

Maybe? Will show us any that have 0 disruptions and we can investigate from there?